script/app.js

```diff
@@ -0,0 +1,106 @@
+//1.написать функцию  filterFalsyValues, которая принимает массив и возвращает массив без '', null, NaN, false, undefined, 0:
+function filterFalsyValues(arr) {
+  return arr.filter(function(el) {
+   if (el === '') {return false;}
+   if (el === null) {return false;}
+   if (el === false){return false;}
+   if (el === undefined) {return false;}
+   if (el === 0) {return false;}
```

Comment:
it can be simplified.
As you remember to keep item in array - you should make return true.
To remove item - you should make 'return false', but all your values are already falsy.
you can just write return item; - will leave only truthy values in array